src/components/Header.jsx

```diff
@@ -1,12 +1,25 @@
 import { Link } from "react-router-dom";
+import { useContext } from "react";
+import { UserContext } from "../contexts/UserContext";
 
 function Header() {
+    const { user } = useContext(UserContext)
+    
     return (
         <div>
             <h1>NC NEWS</h1>
+            {user && user.username ? <p className="user_text">Logged in as: {user.username}</p> : null}
             <Link className='link' to={`/`}>
                 Home
             </Link>
+            <br></br>
```

Comment:
I would recommend you look at ways of handling the spacing between your links with CSS rather than an additional element.